### DISCUSSION TITLE
DONE/SOLVED: Some feature recommendations

### DISCUSSION THREAD

**Discussion by Herodrink** [WowOpenBox/WowOpenBox#73] [Ideas]
I wasn't sure if you are tracking feedback here or in issues.  Before I jump into some feature recommendations, I want to share how I multibox.

Right now I run a 5 Man: Tank(Prot Paladin) Healer(Priest or any really) Caster (3x Warlocks or 2 Warlocks and 1 Mage).  My window layout is typically a large window for the tank, 3 small windows at the corner for the DPS and one wide window (or an always on top window) for the Healer.  The Tank and DPS all use assist castsequence or GSE macros connected to my inifinite scroll wheel on a G602.  So basically, when I start an encounter i setup the paladin with Seals/Judge and then rotate consecrate and holy shield and then spin my mouse wheel down in a circular pattern at the corner of my screen to launch curses, corruptions, shadowbolts, frostbolts etc.  As units take damage I use a really large ElvUI or Grid2 Party Frame (it take up 90% of the healers UI) and Clique or mouseover keybinds to heal.  I use 2 addons EMA Classic and Follow Me.  F1 is key binded to /p FME which initiates the follow me script and EMA Classic is basically a mandatory mod.  F2 is mount.  F3 is interact with the target. F4 is hearthstone.   

It works extremely well.  I find it works a lot better than round-robin and my mouse is only in a small space.  It's not as good as input broadcasting but it's like 99% of the way there.  

I have been evaluating different MB tools since IsBoxer died.  

Normally I use Shift Window + X-Mouse Controls to do this.  Shift Windows sets up my window layout with saved profiles and X-Mouse Control lets me mouse into each window.  

I tried JMB but it's just not ready for primetime in my opinion and most of the devs I have interacted with in Lax's discord are not very friendly.  I keep an eye in there in case there's relevany updates but honestly there aren't.

FurySwipe's 5MMB wouldn't really fit my use case but he's done some great work that I'll mention below.

WOB is by far the best Multibox solution I have some across.  I think a couple of recommendations could easily put it above and beyond other MB solutions.

Here's a couple of recommendations:

Window Layout Profiles- being able to save custom window layout profiles would be huge.  If I spend a bunch of time building a window layout and then leave the app and come back, all my work is lost (I think).  ShiftWindow does this exceptionally well but lacks a visual representation of the windows (it's all screen location data).  Combine the ability to specify Window Layout Profiles and add an import-export profile option.

Mouse Over Functionality
It might have just been me but I couldn't get the mouse over window function to work 100%.  I use X-Mouse Control for that and it works flawlessly.  https://joelpurra.com/projects/X-Mouse_Controls/

FurySwipes Account/Password Data
It takes me a while to input all of my account data.  FurySwipes has an elegant solution for this.  Would be awesome to integrate it.

Window Overlay Selection
The overlay is awesome.  Definitely one of the better solutions out there.  It would be great to have the full border change color when mousing into those windows.

That's all I can think of for now.

**Comment by mooreatv** [comment]
First thanks a lot for taking the time to share your ideas (and the praise!):

- Profiles are already there, probably not so well documented/discoverable - check the File and Profile menus? it's a bit klunky as it saves both default and target profile but it should work - let me know if it doesn't or how it can be improved
- Can you give more details about mouse follow focus issues because it does work flawlessly and also works exactly like xmouse control as far as I know (it's just windows standard settings being turned on)
- I really recommend you use bnet to launch your wow, so you never type any password. I don't think it's safe to do otherwise (specifically 5mbb uses send keys to send password and that's not a good idea at all as that's the banned APIs)
- Adding more feature related to entering/leaving a window is a good idea, will file (there is a similar request #65)

**Comment by I3igAl** [comment]
@Herodrink How are you using scroll wheel up and down in WoW for keybinds? did you unbind camera zoom in game and apply them to bar slots instead? i have been trying to find a way to use them as spell binds but you cannot change scroll up and down in the Logitech software (I have a G500s mouse, cannot let go of my infinite scroll)

**Comment by mooreatv** [comment]
Going to mark this DONE (by editing the subject)

If somethings isn't covered in 4.2.3 can you please file an issue?
